public/index.html

```diff
@@ -20,18 +20,10 @@
   <meta property="twitter:title" content="Twin Cities Aid Distribution Locations">
   <meta property="twitter:description" content="A map to coordinate aid and care in the Twin Cities.">
   <meta property="twitter:image" content="https://twin-cities-mutual-aid.org/images/aid.png">
-
-  <!-- MAPBOX -->
-  <script src="https://api.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.js"></script>
-  <link href="https://api.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.css" rel="stylesheet" />
-  <script src="https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-geocoder/v4.5.1/mapbox-gl-geocoder.min.js"></script>
-  <link rel="stylesheet" href="https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-geocoder/v4.5.1/mapbox-gl-geocoder.css" type="text/css" />
-  <script src="https://cdn.jsdelivr.net/npm/lodash@4.17.15/lodash.min.js"></script>
-
+	
 	<!-- CONFIG -->
 	<script>
-		mapboxgl.accessToken = 'pk.eyJ1IjoiamFjb2JyZGFsdG9uIiwiYSI6ImNrYXc2anFjbjIxNGwyeG14dWk0MDVycmIifQ.yILWplracVMB4mv1dWwTtg'
-		const DATA_URL = 'https://spreadsheets.google.com/feeds/list/1CyPozeKhmOuIaVnKDQAUKKsvA9R4F20hEQ3MSGWczP8/1/public/full?alt=json'
+
 	</script>
```

Comment:
Is this just a little empty script guy now?

Reply:
Looks like it

Reply:
Good catch!